corpus/templates/components/navbar_large.html

```diff
@@ -1,6 +1,9 @@
 <div class="navbar bg-primary text-primary-content sticky top-0 z-10">
     <div class="navbar-start">
-        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Corpus</a>
+        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">
+            <img src="/static/img/ieee-nitk.png" alt="IEEE NITK Logo" class="dark-logo h-10">
+            <img src="/static/img/ieee-nitk-white.png" alt="IEEE NITK Logo" class="light-logo h-10">
```

Comment:
```suggestion
            <img src="{% static 'img/ieee-nitk.png' %}" alt="IEEE NITK Logo" class="dark-logo h-10">
            <img src="{% static 'img/ieee-nitk-white.png' %}" alt="IEEE NITK Logo" class="light-logo h-10">
```

We shouldn't be managing static files with direct paths. In case we choose to migrate static files to a different folder, we would have to reconfigure all the static paths again.

We should instead be using Django's static file manager, which configures the static file path in the template itself (see [here](https://docs.djangoproject.com/en/5.0/howto/static-files/#configuring-static-files)).

Reply:
okay